ui-tests-protractor/e2e/common/common.po.ts

```diff
@@ -0,0 +1,84 @@
+import { SyndesisComponent } from '../common/common';
+import { $, browser, ElementFinder, ExpectedConditions } from 'protractor';
+import { P, World } from '../common/world';
+import { log } from '../../src/app/logging';
+
+export class CommonMethods {
```

Comment:
I would rather rename the class to indicate that it's still related to Datamapper common methods to prevent any further confussion. Lets keep it very specific & targeted for now.